samples/optimizely-audiences/index.html

```diff
@@ -0,0 +1,134 @@
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+</head>
+
+<body>
+  <p class="loading">Loading ...</p>
+  <p class="cf-field-error"></p>
+
+  <div class="cf-form-field">
+    <div class="select-container"></div>
+    <div class="cf-form-hint">
+      Please select the target audience for this content.
+    </div>
+    <div class="optimizely-logo"></div>
+  </div>
+
+
+
+  <script>
+    var cfExt = window.contentfulExtension || window.contentfulWidget;
+
+    cfExt.init(function (api) {
+      api.window.startAutoResizer();
+
+      $.ajax({
+        type: 'GET',
+        beforeSend: function (request) {
+          // Read-only api token for Optimizely test account. @TODO(floelhoeffel): Move to configuration.
+          request.setRequestHeader('Authorization', 'Bearer 2:5pj23ZOmd4X_pTPR50P_gwl3SjBguvmJ8o2gaqDj0DRg842OFUGc');
+        },
+        url: 'https://api.optimizely.com/v2/audiences?project_id=10467324025',
+        success: function (response) {
+
+          // Filter archived audiences.
+          var audiences = response.filter(function (item) {
+            return !item.archived
+          })
+
+          // Hide loader.
+          document.querySelector('.loading').style.display = 'none';
+          // Show form.
+          document.querySelector('.cf-form-field').style.display = 'block';
+
+          createDropDown(audiences);
+        },
+        error: function (response) {
+          showError(response.status + ' ' + response.statusText + ' - ' + response.responseJSON.message);
+        }
+      });
+
+      function showError(msg) {
+        document.querySelector('.loading').style.display = 'none';
+        var errorEl = document.querySelector('.cf-field-error');
+        errorEl.style.display = 'block';
+
+        errorEl.appendChild(document.createTextNode(msg));
+        document.querySelector('.cf-form-field').style.display = 'none';
+      }
+
+      function createDropDown(data) {
+
+        // Map structure of data so it can seed the dropdown
+        var dropdownData = data.map(function (data) {
+          return { 'id': data.id, 'text': data.name };
+        });
+
+        var selectEl = document.createElement('select');
+        selectEl.id = 'audiences-select'
+        selectEl.setAttribute('multiple', 'multiple')
+        document.querySelector('.select-container').append(selectEl);
+        var $select2 = $('#audiences-select').select2({
+          placeholder: 'Select audiences',
+          data: dropdownData
+        })
+
+        $select2.on('change', function (e) {
+          var fieldData = [];
+          $select2.select2('data').forEach(function (item) {
+            fieldData.push(item.id);
+          })
+          api.field.setValue(fieldData)
+        });
+
+        var cleanupHandler = api.field.onValueChanged(function () {
```

Comment:
`cleanupHandler` seems to be unused.

Reply:
I changed so now it uses the same approach from the vanilla template to properly cleanup